### ISSUE TITLE
Filter grafana dashboard on click

### ISSUE THREAD

**Issue by raulsperoni** [raulsperoni/magnesium-wordcloud-panel#5]
It would be really nice to use the library's available callbacks to filter grafana dashboard data when a word is clicked. I haven't found a neat way to do this yet. Related to [this](https://community.grafana.com/t/set-ad-hoc-filter-from-panel-plugin/26741/3).

**Comment by mfishman64** [comment]
I agree.  The current word cloud is a great visualization that we use very frequently.  A clickable word cloud would be a great feature.

**Comment by raulsperoni** [comment]
maybe this is working now: https://community.grafana.com/t/how-can-i-change-template-varibale-in-a-react-plugin-in-grafana-7-0/31345/2?u=raulsperoni 

I will need to test it

**Comment by vvandel-nfir** [comment]
Great wordcloud plugin, I also like how mousing over a word shows you the value. Being able to click on a word for a drill down would make it even better!
